components/IconCookie.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+defineProps({
+    width: {
+        type: Number || String,
+        default: 48
+    },
+    height: {
+        type: Number || String,
+        default: 48
+    },
```

Comment:
doesn't make sense to pass these pros.

You can choose the dimension directly when you use the component like this:
` <icon-cookie class="w-56 h-56"/>`
Because these class will be applied to the first tag of the component that in this case is `<svg>`